File: Praktikum Python/PRAK505-2210817210033-HariOctavianDelrossi.py
def Biodata(tahunLahir, A, B):
    tahun_sekarang = 2020
    tahunLahir = int(tahunLahir)
    print("\nPerkenalkan Nama Saya :", A)
    print("Umur Saya :", tahun_sekarang-tahunLahir)
    print("Saya Adalah Angkatan :", tahun_sekarang)
    print("Asal Saya dari :", B)

tahunLahir = int(input())
A = input()
B = input()
Biodata(tahunLahir, A, B)

File: Praktikum C/PRAK505-2210817210033-HariOctavianDelrossi.c
#include<stdio.h>
void Biodata(int tahunLahir, char A[20], char B[15]){
    int tahun_sekarang = 2020;
    printf("\nPerkenalkan Nama Saya : %s\n", A);
    printf("Umur Saya : %d\n", tahun_sekarang - tahunLahir);
    printf("Saya Adalah Angkatan : %d\n", tahun_sekarang);
    printf("Asal Saya dari : %s", B);
}
int main(){
    int tahunLahir;
    char A[20], B[15];
    scanf("%d",&tahunLahir);
    scanf(" %[^\n]",&A);
    scanf(" %[^\n]",&B);
    Biodata(tahunLahir, A, B);
    return 0;
}